text/warn.md

```diff
@@ -0,0 +1,9 @@
+You are receiving this message because you have broken some rules or are on the verge of doing so.
+
+While we welcome everyone with open arms and discussions of all sorts whether related to Codewars or not, we ask that you adhere to a few basic rules:
+
+- Do not use this server to promote your own products and/or services
+- Do not abuse this server for free help on your programming assignments, homework, exercise, etc. Asking for help and guidance once in a while is fine; expecting to be spoonfed every single time and draining everyone else's energy is not
+- Do not post content which may not be deemed appropriate for a particular age range. This includes, but is not limited to: nudity, violence, racial / gender discrimination, "edgy" memes which may be morally questionable
```

Comment:
Just use the sentences from the server's `#readme`, which is based on what Discord provided:

> 1. Treat everyone with respect. Absolutely no harassment, witch hunting, sexism, racism, or hate speech will be tolerated.
> 2. No NSFW or obscene content. This includes text, images, or links featuring nudity, sex, hard violence, or other graphically disturbing content.
> 3. No spam or self-promotion (server invites, advertisements, etc) without permission from @mods. This includes DMing fellow members.

Reply:
I'm not sure how to word the "spoonfeed" one well. I get what you're trying to say.